extras/kinesis-asl/src/main/java/org/apache/spark/examples/streaming/JavaKinesisWordCountASL.java

```diff
@@ -131,7 +131,7 @@ public static void main(String[] args) {
         for (int i = 0; i < numStreams; i++) {
           streamsList.add(
             KinesisUtils.createStream(jssc, streamName, endpointUrl, checkpointInterval, 
-            InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2())
+            InitialPositionInStream.LATEST, StorageLevel.MEMORY_AND_DISK_2(), null)
```

Comment:
another option is to require a non-null value to tighten up the interface - DefaultAWSCredentialsProvider() would be recommended, but people can pass in their own.

again, please make sure there are no Serializable issues if you go this route.  i remember having issues here.